spring-context/src/test/java/org/springframework/aop/framework/ProxyFactoryBeanTests.java

```diff
@@ -10,7 +10,7 @@
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
+ * See the License for the specific languaspring-context/src/test/java/org/springframework/context/annotation/ReflectionUtilsIntegrationTests.javage governing permissions and
```

Comment:
I'm not quite sure how you managed to modify that, but I'll fix it when merging. 😉

Reply:
Sorry, it should be my hand shaking.😿